packages/fuels-test-helpers/src/node.rs

```diff
@@ -42,7 +41,7 @@ impl Config {
             predicates: false,
             manual_blocks_enabled: false,
             vm_backtrace: false,
-            silent: true,
+            silent: false, // Temp for debugging.
```

Comment:
Is this intentional?

Reply:
Nope, that was for debugging the fuel-core binary behaviour inside the CI :-)